boilerplate/app/components/text/text.tsx

```diff
@@ -12,10 +12,10 @@ import { mergeAll, flatten } from "ramda"
  */
 export function Text(props: TextProps) {
   // grab the props
-  const { preset = "default", tx, text, children, style: styleOverride, ...rest } = props
+  const { preset = "default", tx, txOptions, text, children, style: styleOverride, ...rest } = props
 
   // figure out which content to use
-  const i18nText = tx && translate(tx)
+  const i18nText = tx && translate(tx, txOptions)
   const content = i18nText || text || children
```

Comment:
could maybe be one-liner? 
```
const content = tx ? translate(tx, txOptions) : text ? text : children
```

Reply:
Nested ternaries aren't great for readability.

Reply:
@yulolimum @kevinvangelder I was going to say not everyone loves nested ternaries as much as you and I.

Reply:
I agree with Kevin here; although the nested ternary is clever, it's less readable, and we have non-IR peeps trying to read this code.